server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/DeletedMessagesVaultTest.java

```diff
@@ -483,7 +483,9 @@ private void restoreAllMessagesOfHomer() {
 
     private void restoreMessagesFor(String user) {
         String taskId = webAdminApi.with()
-            .post("/deletedMessages/user/" + user + "?action=restore")
+            .queryParam("action", "restore")
+            .body("{\"criterions\":[]}")
+            .post("/deletedMessages/user/" + user).prettyPeek()
```

Comment:
`prettyPeek()` ?

Reply:
I'm NEVER letting debug statement make it to the review... :trollface: